doc/source/packaging/structure.rst

```diff
@@ -98,7 +154,8 @@ use of `Sphinx`_ requires the following structure, as shown by figure
 The ``source/`` directory needs to contain at least the following files:
 
 - ``conf.py`` is a Python script used to declare the configuration of `Sphinx`_.
-- ``index.rst`` is the index page of the documentation.
+- ``index.rst`` is the index page of the documentation. Try to reuse the
+  ``README.rst`` file in here to avoid duplication.
```

Comment:
```suggestion
- ``index.rst`` is the index page of the documentation. In this file, try to reuse the
  ``README.rst`` file to avoid duplication.
```